seawatch_registration/templates/seawatch_registration/registration_process.html

```diff
@@ -8,6 +8,44 @@
 
     <h1>{{ view.title }}</h1>
 
+    <div class="row">
```

Comment:
The for loop over the registration steps should capture Profile, Additional Questions and Requested Positions.

Reply:
oh this wasnt supposed to get on this brunch. Here PyCharm didn't merge the way i wanted it to merge. I will fix it.

Reply:
Classic.